mobile/android/android-device.ts

```diff
@@ -259,6 +259,9 @@ export class AndroidDevice implements Mobile.IAndroidDevice {
         var clearDebugEnvironmentCommand = this.composeCommand('shell rm "%s"', envDebugOutFullpath);
         this.$childProcess.exec(clearDebugEnvironmentCommand).wait();
 
+		var setDebugBreakEnvironmentCommand = this.composeCommand('shell echo "" > "%s"', "debugbreak");
```

Comment:
```'shell echo "" > "%s"', "debugbreak"```
vs
```"shell touch debugbreak"```

Looks much more readable to me. Furthermore, man touch explicitly says: "If any file does not exist, it is created ..." while using echo is a hack.

My justification is that I suggest using the right tool for the purpose, which as a side benefit produces shorter and easier to read and support code. 

Of course, your code may depend on the created file containing an empty line, in which case, echo is the correct approach and you can disregard my comment.